src/plugins/base64/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
+include (CheckSymbolExists)
 include (LibAddPlugin)
 
+check_symbol_exists (strdup "string.h" HAVE_STRDUP)
+if (NOT HAVE_STRDUP)
+	remove_plugin(base64 "requires strdup")
+endif (NOT HAVE_STRDUP)
```

Comment:
Better fix: use elektraStrDup

Reply:
So far I have not done any changes to the 'code' itself. Just getting auto exclusion getting to work will be massive.

Reply:
#2097